contract-sdk/Cargo.toml

```diff
@@ -10,12 +10,16 @@ license = "Apache-2.0"
 cbor = { version = "0.2.1", package = "oasis-cbor" }
 oasis-contract-sdk-types = { path = "types" }
 oasis-contract-sdk-macros = { path = "../contract-sdk-macros", optional = true }
+oasis-contract-sdk-crypto = { path = "crypto" }
```

Comment:
Should this only be a dependency on non-wasm32 since it is only used in tests? E.g.
```toml
[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
oasis-contract-sdk-crypto = { path = "crypto" }
```